src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -48,21 +65,102 @@ public String getFeedSourceId () {
     @Override
     public void jobLogic () {
         LOG.info("Processing feed for {}", feedVersion.id);
+        feedSource = feedVersion.parentFeedSource();
+        // If there are any feed transformations that apply to the VERSION_CLONE retrieval method, we need to add a
+        // stage to jobFinished (perhaps) to clone the feed version (as long as this feed version has not already been
+        // cloned). Once that feed version is processed, the appropriate transformations will apply themselves.
+        shouldClone = !feedVersion.retrievalMethod.equals(VERSION_CLONE) &&
+            feedSource.hasRulesForRetrievalMethod(VERSION_CLONE);
```

Comment:
Are there any Feed Transformations that apply to version clones? I'm not sure if I found any. Therefore, it's kinda hard to see the purpose of this.